Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y \
+    curl \
+    wget \
+    git-core \
+    gcc \
+    g++ \
+    cmake \
+    python \
+    python2.7 \
+    python2.7-dev \
+    zlib1g-dev \
+    bzip2 \
+    libyaml-dev \
+    libyaml-0-2
+RUN wget https://raw.github.com/pypa/pip/master/contrib/get-pip.py -O - | python
```

Comment:
I believe there is an apt-get package called `python-pip` or similar that can be used instead of this.